ygot/pathstrings.go

```diff
@@ -234,6 +234,16 @@ func StringToStructuredPath(path string) (*gnmipb.Path, error) {
 	return gpath, nil
 }
 
+// MustStringToPath calls StringToStructuredPath and panics on error.
+// It is intended for use in tests with hard-coded strings.
+func MustStringToPath(path string) *gnmipb.Path {
```

Comment:
Moved to testutil.

Reply:
Actually, this is a problem -- it creates a circular dependency because `ygot` depends on `testutil`. The standard library uses this Must pattern, e.g. https://pkg.go.dev/regexp#MustCompile

I think this is ok since it's widely used in the standard library, what do you think?

Reply:
I've split this code to https://github.com/openconfig/ygot/pull/775 for easier review.